content/docs/command-reference/exp/index.md

```diff
@@ -0,0 +1,48 @@
+# exp[eriments]
+
+A set of commands to generate and compare local _experiments_:
+[show](/doc/command-reference/exp/show),
+[checkout](/doc/command-reference/exp/show),
+[diff](/doc/command-reference/exp/diff), [run](/doc/command-reference/exp/run),
```

Comment:
Same, run before diff, prob. Maybe also resume before diff, even, since I think it's closely linked to run.